tests/db.test.js

```diff
@@ -0,0 +1,41 @@
+const db = require('../database/index.js')
+
+describe('Database', () =>{
+
+  it('should retrive all records in the database', (done) => {
+    db.client.query(`SELECT * from recipes`, (err, res) => {
+      if(err) {
+        throw err
+      } else {
+        var results = res.rows.length;
+        console.log(results);
+        expect(results).toBe(101);
+        done();
+      }
+    });
+  });
+  it('should return a single record when given an id', (done) => {
+    db.client.query(`SELECT * from recipes where id = 9`, (err, res) => {
+      if(err) {
+        throw err
+      } else {
+        var record = res.rows[0].id;
+        console.log(record);
+        expect(record).toBe(9);
+        done();
+      }
+    });
+  })
+  it('should return undefined if the record id does not exist', (done) => {
+    db.client.query(`SELECT * from recipes where id = 200`, (err, res) => {
+      if(err) {
+        var error = err
```

Comment:
I don't think this is right here

Reply:
@CareyL96 Made the changes you request, awaiting your review. Thanks for your diligence. Much appreciated.